ansible/roles/chroma/tasks/main.yaml

```
@@ -0,0 +1,108 @@
+---
+- name: "Create user for chroma"
+  user:
+    name: chroma
+    home: "/var/www/chroma"
+    shell: "/usr/sbin/nologin"
+    system: true
+    state: "present"
+
+- name: "Create chroma frontend directory"
+  file:
+    name: "/var/www/chroma/frontend"
+    owner: "chroma"
+    group: "chroma"
+    mode: "2775"
+    state: "directory"
+  
+- name: "Create chroma docs directory"
+  file:
+    name: "/var/www/chroma/docs"
+    owner: "chroma"
+    group: "chroma"
+    mode: "2775"
+    state: "directory"
+
+- name: "Create chroma server directory"
+  file:
+    name: "/var/www/chroma/server"
+    owner: "chroma"
+    group: "chroma"
+    mode: "2775"
+    state: "directory"
+
+- name: "Download and extract chroma frontend"
+  unarchive:
+    src: "https://github.com/svsticky/chroma/releases/download/1.0/frontend.tar.gz"
+    dest: "/var/www/chroma/frontend"
+    remote_src: yes
+    owner: "chroma"
+    group: "chroma"
+    mode: "775"
+
+- name: "Download and extract chroma docs"
+  unarchive:
+    src: "https://github.com/svsticky/chroma/releases/download/1.0/docs.tar.gz"
+    dest: "/var/www/chroma/docs"
+    remote_src: yes
+    owner: "chroma"
+    group: "chroma"
+    mode: "775"
+
+- name: "Download and extract chroma server"
+  unarchive:
+    src: "https://github.com/svsticky/chroma/releases/download/1.0/server-x86_64-unknown-linux-musl"
+    dest: "/var/www/chroma/server"
+    remote_src: yes
+    owner: "chroma"
+    group: "chroma"
+    mode: "777"
```

I just realized we can cut the `dist` folder by setting
```suggestion
    mode: "777"
    extra_opts:
      - "--strip-components=1"  
```
like we also did here: https://github.com/svsticky/sadserver/blob/42fc9aa819095303d49f5e0bae3cb6ba234a3190/ansible/roles/oauth2_proxy/tasks/main.yml#L21